draft-ietf-quic-http.md

```diff
@@ -272,11 +272,12 @@ fully closed.
 A server can send a complete response prior to the client sending an entire
 request if the response does not depend on any portion of the request that has
 not been sent and received. When this is true, a server MAY request that the
-client abort transmission of a request without error by sending a RST_STREAM
-with an error code of NO_ERROR after sending a complete response and closing its
-stream. Clients MUST NOT discard responses as a result of receiving such a
-RST_STREAM, though clients can always discard responses at their discretion for
-other reasons.
+client abort transmission of a request without error by triggering a QUIC
+DISINTEREST with error code HTTP_EARLY_RESPONSE, sending a complete response,
```

Comment:
We need to reconcile this error code business, otherwise, I think we're good to go.  (And I would like to resolve this soon.)